src/main/java/features/Storage.java

```diff
@@ -0,0 +1,92 @@
+package features;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+import tasks.Deadline;
+import tasks.Event;
+import tasks.ToDo;
+
+/**
+ * Saves and loads state of taskList to/from data/duke.txt.
+ */
+public class Storage {
+
+    /**
+     * Loads taskList from data/duke.txt, if any.
+     * @return the loaded taskList.
+     */
+    public TaskList loadTaskList() {
+
+        TaskList toLoad = new TaskList();
```

Comment:
`toLoad` sounds a bit vague on what this variable does. `newTaskList` for example sounds more precise.

Reply:
Thanks, i've fixed this now.